bin/yarn.js

```diff
@@ -30,7 +30,8 @@ if (semver.satisfies(ver, '>=5.0.0')) {
   possibles.push('../updates/current/lib-legacy/cli/index.js');
   possibles.push('../lib-legacy/cli/index.js');
 } else {
-  throw new Error('Node version ' + ver + ' is not supported');
+  console.error('\033[31m', 'Node version ' + ver + ' is not supported, please use `v>=4`.' ,'\033[31m');
```

Comment:
What about changing this to "please use Node.js 4.0 or higher"? "v>=4" is a little too terse and not user-friendly 😄